cblite/llm/Bedrock.cc

```diff
@@ -0,0 +1,60 @@
+//
+// Bedrock.cc
+//
+// Copyright (c) 2024 Couchbase, Inc All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "Bedrock.hh"
+
+using namespace std;
+using namespace fleece;
+using namespace litecore;
+
+// LiteCore Request and Response
+alloc_slice Bedrock::run(const string& restBody, C4Error error) {
+    Encoder enc;
+    enc.beginDict();
+    enc["Content-Type"_sl] = "application/json";
+    enc["Content-Length"_sl] = restBody.length();
+    if (getenv("API_KEY") == NULL) {
```

Comment:
I think this env variable should be changed to something a bit more descriptive as well.  Perhaps LLM_API_KEY or ENRICH_API_KEY to make sure it doesn't clash with something else in the environment.